code/obj/item/organs/eye.dm

```diff
@@ -464,12 +467,42 @@ TYPEINFO(/obj/item/organ/eye/cyber/laser)
 		else // just us!
 			aholder.removeAbility(abil)
 
+TYPEINFO(/obj/item/organ/eye/cyber/monitor)
+	mats = 7
+
+/obj/item/organ/eye/cyber/monitor
+	name = "monitor cybereye"
+	organ_name = "monitor cybereye"
+	desc = "A tiny screen to replace an eye. You can't use it to see, but it can view some camera networks."
+	organ_abilities = list(/datum/targetable/organAbility/view_camera)
+	default_material = "pharosium"
+	iris_color = "#0d0508"
+	icon_state = "eye-monitor"
+	provides_sight = FALSE
+	var/obj/item/device/camera_viewer/viewer = null
+
+	New()
+		. = ..()
+		src.viewer = new(src)
+		src.viewer.network = list("public", "Robots", "telesci")
```

Comment:
Robots will include syndicate cyborgs and any in upload (allowing basically free checking of upload if you know a borg is in there. I think this should be restricted to the emag effect.